test/verify/check-pages

```diff
@@ -319,8 +319,7 @@ OnCalendar=daily
             })""", language)
             b.wait_attr(".index-page", "lang", language)
 
-        # the quick iteration starts/stops tracer in quick succession, before it can finish
-        self.allow_browser_errors("Tracer failed:.*internal-error")
+        self.allow_restart_journal_messages()
```

Comment:
Do tests use `allow_restart_journal_messages` also for logging in/out? I'd say it is meant for restarting cockpit-ws or rebooting the machine... It's probably not super well defined.

Reply:
Yes, they do. That's how we've used that in the recent years. Even on regular logouts there's so many processes and watches that are being killed, that's going to make some noise.